.clang-tidy

```diff
@@ -29,19 +39,24 @@ hicpp-avoid-goto,
 misc-unused-alias-decls,
 misc-unused-using-decls,
 modernize-*,
+-modernize-avoid-c-arrays,
 -modernize-concat-nested-namespaces,
+-modernize-macro-to-enum,
 -modernize-return-braced-init-list,
 -modernize-use-auto,
 -modernize-use-default-member-init,
 -modernize-use-using,
 -modernize-use-trailing-return-type,
 -modernize-use-nodiscard,
+-modernize-redundant-void-arg,
```

Comment:
Huh, what's wrong with this check?

Reply:
This is a good check but too many callback functions or lambda use (void)